src/components/central-programs-table.js

```diff
@@ -240,10 +166,93 @@ const ModalColumnToggle = ({
 }
 
 
-const CentralProgramsTable = ({data}) => {
+const CentralProgramsTable = ({data, labelContent}) => {
   const firstRow = createFirstRow(data)
+
+  //per-sort column data so we don't need to search the array every time
+  let columnLabelsByDatafield = {}
+  labelContent.columns.forEach(c => {
+    columnLabelsByDatafield[c.dataFieldName] = c
+  })
+
   // creates new array
   data = data.concat([firstRow])
+
```

Comment:
moving columns into the component so they can access the dynamic translation information that's being passed in